### PR TITLE
JAMES-3462: CassandraMailboxChangeRepository implementation

### PR DESCRIPTION
**Pull request by LanKhuat**

### CODE REVIEW

File: server/data/data-jmap-cassandra/src/main/java/org/apache/james/jmap/cassandra/change/CassandraMailboxChangeModule.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import static org.apache.james.jmap.cassandra.change.tables.CassandraMailboxChangeTable.IS_DELEGATED;
+import static org.apache.james.jmap.cassandra.change.tables.CassandraMailboxChangeTable.STATE;
+import static org.apache.james.jmap.cassandra.change.tables.CassandraMailboxChangeTable.TABLE_NAME;
+import static org.apache.james.jmap.cassandra.change.tables.CassandraMailboxChangeTable.UPDATED;
+
+import org.apache.james.backends.cassandra.components.CassandraModule;
+
+public interface CassandraMailboxChangeModule {
+    CassandraModule MODULE = CassandraModule.table(TABLE_NAME)
+        .comment("Holds MailboxChange definition.")
```
Comment (chibenwa) [inline server/data/data-jmap-cassandra/src/main/java/org/apache/james/jmap/cassandra/change/CassandraMailboxChangeModule.java:46]: Holds MailboxChange definition. Used to manage Mailbox state in JMAP.

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
```
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] Caused by: com.datastax.driver.core.exceptions.InvalidQueryException: unconfigured table mailbox_change
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.datastax.driver.core.Responses$Error.asException(Responses.java:181)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.datastax.driver.core.SessionManager$4.apply(SessionManager.java:250)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.datastax.driver.core.SessionManager$4.apply(SessionManager.java:219)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.google.common.util.concurrent.AbstractTransformFuture$AsyncTransformFuture.doTransform(AbstractTransformFuture.java:210)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.google.common.util.concurrent.AbstractTransformFuture$AsyncTransformFuture.doTransform(AbstractTransformFuture.java:199)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.google.common.util.concurrent.AbstractTransformFuture.run(AbstractTransformFuture.java:111)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.google.common.util.concurrent.MoreExecutors$5$1.run(MoreExecutors.java:992)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at com.datastax.shaded.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] 	at java.base/java.lang.Thread.run(Thread.java:834)
[fe9b64f9eac252c6ee5026f54dd31b3096018a95] ">
```

You are missing a binding for the Cassandra module

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
test this please

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**